modules/registrars/synergywholesaledomains/synergywholesaledomains.php

```diff
@@ -283,7 +283,13 @@ function synergywholesaledomains_getConfigArray(array $params)
         'defaultDnsConfig' => [
             'FriendlyName' => 'Default DNS Config',
             'Type' => 'dropdown',
-            'Options' => ['Nothing', 'Names Servers', 'FreeDNS with email forwarding', 'Parked', 'FreeDNS'],
+            'Options' => [
+                0 => 'Nothing',
+                1 => 'Names Servers',
```

Comment:
Can we please change this to Nameservers without the space? Mostly just to keep consistent with other messages in WHMCS, Synergy and in this module.

Reply:
Sounds good to me, I copied this from the task requirements